### PR TITLE
feat: add salesforce pub sub to chart

### PR DESCRIPTION
**Pull request by lucasmarshall**

### CODE REVIEW

File: charts/supaglue/templates/_helpers.tpl [hunk 1]
```diff
@@ -137,6 +137,29 @@ and we want to make sure that the component is included in the name.
 {{- end -}}
 {{- end -}}
 
+{{- define "supaglue.salesforcePubSub.databaseUrl" -}}
```
Comment (tomkit) [inline charts/supaglue/templates/_helpers.tpl:140]: Can you leave a comment this is experimental, alpha, or for salesforce pubsub only

File: charts/supaglue/templates/salesforce-pub-sub-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
+{{- if $.Values.salesforcePubSub.enabled }}
```
Comment (tomkit) [inline charts/supaglue/templates/salesforce-pub-sub-deployment.yaml:1]: Can you leave a comment this is experimental, alpha, or for salesforce pubsub only

File: charts/supaglue/templates/sync-worker-deployment.yaml [hunk 3]
```diff
@@ -73,7 +73,7 @@ spec:
             - name: DISABLE_BANNER
               value: "1"
             - name: SUPAGLUE_LOG_LEVEL
-              value: {{ .Values.api.logLevel }}
+              value: {{ .Values.syncWorker.logLevel }}
```
Comment (tomkit) [inline charts/supaglue/templates/sync-worker-deployment.yaml:76]: were sync-worker log levels not set correctly before?

File: charts/supaglue/templates/_helpers.tpl (hunk 1 above)
Comment (lucasmarshall) [reply charts/supaglue/templates/_helpers.tpl:140]: The template is called `supaglue.salesforcePubSub.databaseUrl`

Do we not think that is sufficient to call it out as for salesforce pub sub?

File: charts/supaglue/templates/salesforce-pub-sub-deployment.yaml (hunk 2 above)
Comment (lucasmarshall) [reply charts/supaglue/templates/salesforce-pub-sub-deployment.yaml:1]: The file name is `salesforce-pub-sub-deployment.yaml`. Do we not think that is sufficient to call it out as for salesforce pub sub?

File: charts/supaglue/templates/sync-worker-deployment.yaml (hunk 3 above)
Comment (lucasmarshall) [reply charts/supaglue/templates/sync-worker-deployment.yaml:76]: No, they were using the API ones.

**Review by asdfryan — APPROVED** [review]

**Pull request merged**